src/api/todos.ts

```diff
@@ -0,0 +1,23 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+interface Data {
+  completed?: boolean;
+  title?: string;
+}
```

Comment:
Use `Partial` type as data have to include fields from Todo only.
https://www.typescriptlang.org/docs/handbook/utility-types.html